app/resources/css-default.template

```diff
@@ -177,11 +177,16 @@ div.languages {
   text-decoration: none;
   color: black;
   text-align: {{end}};
+  float:right;
+  margin-right:5%;
```

Comment:
We usually don't put a margin with "%", I believe. How does it look with no margin?